sklearn/feature_extraction/tests/test_text.py

```diff
@@ -1653,3 +1653,24 @@ def test_vectorizers_do_not_have_set_output(Estimator):
     """Check that vectorizers do not define set_output."""
     est = Estimator()
     assert not hasattr(est, "set_output")
+
+
+@pytest.mark.parametrize("csr_container", CSR_CONTAINERS)
+def test_tfidf_transformer_copy(csr_container):
+    """Check the behaviour of TfidfTransformer.transform with the copy parameter."""
+    X = sparse.rand(10, 20000, dtype=np.float64, random_state=42)
+    X_csr = csr_container(X)
+
+    # keep a copy of the original matrix
+    X_csr_original = X_csr.copy()
+
+    transformer = TfidfTransformer().fit(X_csr)
+
```

Comment:
```suggestion

# check that we transform on a copy
```

Reply:
Usually, I learnt that we should not have to put this type of comment because the code should be self explicit. The fact that we pass `copy=False/True` is giving this information.

Reply:
I understand, but comments like that one do help less experienced people to grasp the concepts faster and contribute to the project.